platform-designs/design-feedback/README.md

```diff
@@ -9,3 +9,5 @@ For this folder, "design" refers to the aesthetics and design system of the plat
 
 # Feedback Documentation 
 > related issue: [Issue #560](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/560)
+* You can find the feedback summary [here](https://docs.google.com/spreadsheets/d/1I2md6PZjEabZrCnC7dHiXzh1FYU6gCi_v3bU5ubvi5w/edit#gid=475471032). 
+* You can find the analysis of the feedback summary [here](https://docs.google.com/presentation/d/1VTJbexcdFIkcNOYoW9C61iOVWgRw46ec08PSIv9flPs/edit#slide=id.gf192724d12_0_5). The analysis is based on feedback from [18th September](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/focus-groups/20190918_summary-18-September-2019.md#3-platform-design), [24th September](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/focus-groups/20190924_summary-24-September-2019.md#3-platform-design), [4th October](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/focus-groups/20191004_summary-4-October-2019.md#3-platform-design), [user tests for the previous platform](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/community-recommendations/user-tests), and [Figma prototype](https://www.figma.com/file/HxqTSdeyAUNMualnBV0PET/Autspace-(WIP)?node-id=0%3A1). 
```

Comment:
Hi Soledad. 

Thanks for adding these to the README. 

Can you separate out the two statments into two different lines? 

E.g. 

* You can find the analysis of the feedback summary [here](https://docs.google.com/presentation/d/1VTJbexcdFIkcNOYoW9C61iOVWgRw46ec08PSIv9flPs/edit#slide=id.gf192724d12_0_5). 

The analysis is based on feedback from [18th September](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/focus-groups/20190918_summary-18-September-2019.md#3-platform-design), [24th September](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/focus-groups/20190924_summary-24-September-2019.md#3-platform-design), [4th October](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/focus-groups/20191004_summary-4-October-2019.md#3-platform-design), [user tests for the previous platform](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/community-recommendations/user-tests), and [Figma prototype](https://www.figma.com/file/HxqTSdeyAUNMualnBV0PET/Autspace-(WIP)?node-id=0%3A1).

Reply:
Change made.